### ISSUE TITLE
Configure custom mimetype for ttml

### ISSUE THREAD

**Issue by cksachdev** [thecoshman/http#111]
I need to serve ttml files and trying to configure http. In IIS, I can configure like below:
```
<system.webServer>
    <staticContent>
      <remove fileExtension=".vtt" />
      <mimeMap fileExtension=".vtt" mimeType="text/vtt" />
      <remove fileExtension=".ttml" />
      <mimeMap fileExtension=".ttml" mimeType="application/ttml+xml" />
    </staticContent>
</system.webServer>
```
Looking into the modules mime_guess is used and couldn't find any configuration to supply mimetype for certain extension. Is it possible to configure it? 

Commit with mimetype support: https://github.com/thecoshman/http/commit/4d80852c36cfc58b16bd2905ac0e273623fe9af7

**Comment by nabijaczleweli** [comment]
it currently isn't, but I see no reason it couldn't be; i'll see what mistake of an ABI i can conjure up

**Comment by cksachdev** [comment]
Tried same thing with node http-server. Here is the PR link - https://github.com/http-party/http-server/pull/472/files

**Comment by nabijaczleweli** [comment]
@cksachdev can you try with the new `-m/--mime-type` option from `develop` branch (7e50b518c2e281906bfc8e0806cb0a0b2caa8c59 at a minimum)?

**Comment by nabijaczleweli** [comment]
Released in [`v1.10.0`](https://github.com/thecoshman/http/releases/tag/v1.10.0), but feedback still appreciated
